.travis.yml

```diff
@@ -3,8 +3,10 @@ language: node_js
 node_js:
   - "0.10"
   - "0.12"
-  - "4.1"
-  - "5"
+  - 4
+  - 5
+  - 6
+  - 7
```

Comment:
Please make `7` and `8` allowed failures for TravisCI. They will not officially be supported until `node@8` goes LTS in October.